### DISCUSSION TITLE
关于离线问题

### DISCUSSION THREAD

**Discussion by avllin** [zouyaoji/vue-cesium#552] [Q&A]
大佬好，请问下，我通过npm安装，现在想实现离线模式。我查了一下很多地方都引入了 “https://unpkg.com/cesium@latest/Build/Cesium/Cesium.js” 和 “https://unpkg.com/cesium@latest/Build/Cesium/Cesium.css”

 想咨询下大佬，我怎么做可以实现离线，让项目正常运行。star已点。万分感谢。

**Comment by zouyaoji** [comment]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/quickstart

cesiumPath改成你局域网的Cesium.js
